tools/enso4igv/src/main/java/org/enso/tools/enso4igv/EnsoDataObject.java

```diff
@@ -110,4 +140,20 @@ public static MultiViewEditorElement createEditor(Lookup lkp) {
         return new MultiViewEditorElement(lkp);
     }
 
+    private void registerTruffleMimeType(String mime) throws IOException {
```

Comment:
Modern name for MIME type is [Media type](https://en.wikipedia.org/wiki/Media_type)

Reply:
NetBeans terminology is old fashioned.